src/pages/Drinks.jsx

```diff
@@ -1,10 +1,12 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import Header from '../components/Header';
 import SearchBar from '../components/SearchBar';
 
 function Drinks({ match }) {
   return (
     <div>
+      <Header title="Bebidas" match={ match } />;
       <button type="button" data-testid="search-top-btn">click</button>
```

Comment:
```suggestion
      <Header title="Bebidas" match={ match } />
```
Este button click você já fez dentro do componente Header, portanto, não é necessário ele estar aqui. Ok?